app/src/main/res/values/strings.xml

```diff
@@ -186,6 +187,7 @@
     <string name="pref_summary_colored_navigation_bar">Colors the navigation bar in the primary color.</string>
     <string name="pref_summary_colored_app_shortcuts">Colors the app shortcuts in the primary color.</string>
     <string name="pref_summary_audio_ducking">Notifications, navigation etc.</string>
+    <string name="pref_summary_synchronized_lyrics_show">Show synchronized lyrics for song if available</string>
```

Comment:
It should match `*<filename>*.(lrc|txt)` or `*<title>*.(lrc|txt)`.
Default charset on Android should be utf-8 or utf-16 so I'm not sure if I need to set charset explicitly. It seems to work fine with utf-8 files with non-ASCII characters.